cpp/example_code/rds/getting_started_with_db_instances.cpp

```diff
@@ -0,0 +1,1083 @@
+                              const Aws::RDS::RDSClient &client);
+
+        //! Test routine passed as argument to askQuestion routine.
+        /*!
+         \sa testForEmptyString()
+         \param string: A string to test.
+         \return bool: True if empty.
+         */
+        bool testForEmptyString(const Aws::String &string);
+
+        //! Command line prompt/response utility function.
+        /*!
+         \\sa askQuestion()
+         \param string: A question prompt.
+         \param test: Test function for response.
+         \return Aws::String: User's response.
+         */
+        Aws::String askQuestion(const Aws::String &string,
+                                const std::function<bool(
+                                        Aws::String)> &test = testForEmptyString);
+
+        //! Command line prompt/response for yes/no question.
+        /*!
+         \\sa askYesNoQuestion()
+         \param string: A question prompt expecting a 'y' or 'n' response.
+         \return bool: True if yes.
+         */
+        bool askYesNoQuestion(const Aws::String &string);
+
+        //! Command line prompt/response utility function for an int result confined to
+        //! a range.
+        /*!
+         \sa askQuestionForIntRange()
+         \param string: A question prompt.
+         \param low: Low inclusive.
+         \param high: High inclusive.
+         \return int: User's response.
+         */
+        int askQuestionForIntRange(const Aws::String &string, int low,
+                                   int high);
+
+        //! Routine which converts a string of ints to a vector of ints.
+        /*!
+         \sa splitToInts()
+         \param string: A string of ints.
+         \param delimiter: Delimiter between the ints.
+         \return vector<int>: Vector of ints.
+         */
+        std::vector<int> splitToInts(const Aws::String &string,
+                                     char delimiter) {
+            std::vector<int> result;
+            std::stringstream stringStream(string);
+            Aws::String split;
+            while (std::getline(stringStream, split, delimiter)) {
+                try {
+                    result.push_back(std::stoi(split));
+                }
+                catch (const std::exception &e) {
+                    std::cerr << "askQuestionForIntRange error " << e.what()
+                              << std::endl;
+                }
+
+            }
+
+            return result;
+        }
+
+        //! Utility routine to print a line of asterisks to standard out.
+        /*!
+         \\sa printAsterisksLine()
+        \return void:
+         */
+        inline void printAsterisksLine() {
+            std::cout << std::setfill('*') << std::setw(88) << " "
+                      << std::endl;
+        }
+    } // RDS
+} // AwsDoc
+
+// snippet-start:[cpp.example_code.rds.get_started_instances]
+//! Routine which creates a Relational Database Service (Amazon RDS)
```

Comment:
Here and elsewhere, check for: 
a Relational Database Service (Amazon RDS) --> an Amazon Relational Database Service (Amazon RDS)